projects/kit/components/calendar-range/calendar-range.component.ts

```diff
@@ -125,12 +137,6 @@ export class TuiCalendarRange implements OnInit, OnChanges {
         this.selectedPeriod = period;
     }
 
-    public ngOnChanges(): void {
-        if (!this.value) {
-            this.initDefaultViewedMonth();
-        }
-    }
-
```

Comment:
When inputs (for example `defaultViewedMonth`) updated, we need to run `this.initDefaultViewedMonth()`. Not only `value` input